### ISSUE TITLE
Wildy hunter uses the misc gear tab 

### ISSUE THREAD

**Issue by sn0w1753** [oldschoolgg/oldschoolbot#2808]
## 🚀 Feature Proposal

After the inclusion of the wildy gear tab, hunter (specifically black salamander), still bases its gear from the misc gear tab. I know there’s an incredibly low percentage chance of being pk’d while doing wildy hunter, as I went 200m hunter exp doing black chins on osb, but wouldn’t the new wildy tab be more relevant to wildy slayer than misc? Even though the tab was designed for pvm in the wilderness with a chance to lose all, could this tab be used instead of misc? 

Secondly, I’d also like to propose on top of the defensive stats used while in the wilderness, could there be a chance to include other gear slots in the wildy tab into the roll of being pk’d?  

For instance, if you roll the chance of encountering a pker, then your offensive or defensive stats for the wildy gear tab would be rolled.  If your defensive stats are greater than your offensive stats, then you roll a chance at evading the pker or getting sacked [assuming you’re not already skulled and with an assumption that you’re not going to be skull tricked as you have player attack options off by choosing to be more tanky].
 If your offensive stats are greater than your defensive, then you roll a chance at anti-pking the encounter, running the pk encounter off, dying to the pker, or being skull tricked (if not already skulled).  
 
Whether or not you choose to be defensive or offensive in your pk encounter, the same mechanics would work as far as losing items in the wildy tab upon pvp death; however, if you’re choosing to roll with defense, your food and potions consumption would be double the typical encounter of what is currently given. While if you choose to be more offensive, then by rolling the chance to anti pk, you would “scare off” any future pkers for an allotted amount of trips to the wildy. If you “run the pker off” then you have half the allotted time of “scaring off” future pkers as you would by anti pking. If you die to the pker, you would lose the gear and the current set amount of food and pots that would be used in a typical wildy hunter death (x10 brews, x5 rests, x10 shark or whatever). However, if you’re skull tricked, this would come with a heavy downside of not only losing your gear and all pots/food, but at a 5-10m gp loss on top of everything as that would be an assumed loss of trying to anti pk instead of running defensive. 

Effectively, the differences would be 
Choose to tank pk encounters, lose 2x the amount of resources as what you do now. Choose to anti pk and only run off pker, get a guaranteed free XX skilling/pvm trips with no pk encounters and half current rate of lost food/pots on a pvp death. 
Choose to anti pk and successfully kill pker, the guaranteed free XX skilling/pvm trips with no pk encounters would be doubled without any loss of resources. 
Choose to anti pk and get skull tricked, you lose double the resources (as you would choosing to tank and were to die) and 5-10m based on a fixed rate or variable based on bank value. 
If you don’t have however much gp that could be lost whether choosing defensive or offensive, then you can’t enter wildy just as you can’t do kg or enter rev caves in the main game. 

** TLDR**
Just figured I’d pitch an idea or something that could spark some ideas along with my issue of wildy skilling not basing gear on the wildy tab

**Comment by sn0w1753** [comment]
Ty gidid
